datadog_checks_dev/datadog_checks/dev/tooling/constants.py

```diff
@@ -25,9 +26,12 @@
     ('beta', lambda v: semver.bump_prerelease(v, 'beta')),
 ])
 
-# The checks requirement file used by the agent
-AGENT_REQ_FILE = 'requirements-agent-release.txt'
-
+# The requirements file listing integrations to be included in the Agent package
+AGENT_RELEASE_REQ_FILE = 'requirements-agent-release.txt'
+# The requirements file listing dependencies needed by the embedded Python environment
+AGENT_REQUIREMENTS = os.path.join(
```

Comment:
Since this file get's loaded upon any command let's make this a function that returns `os.path.join(get_root(), ...)` to avoid increasing start-up overhead

Reply:
I don't get it.. I haven't added the root to the path for simmetry with `AGENT_RELEASE_REQ_FILE`

Reply:
I mean if this was `time.sleep(60)` every command would take at least a minute (even `--help`) but if we put it in a function only the callers would

Reply:
Fixed